src/libs/TransactionUtils.ts

```diff
@@ -360,6 +361,13 @@ function getValidWaypoints(waypoints: WaypointCollection, reArrangeIndexes = fal
     }, {});
 }
 
+/**
+ * Returns the most recent transactions in an object
+ */
+function getRecentTransactions(transactions: Record<string, string>, size = 2): unknown[] {
+    return _.sortBy(Object.keys(transactions), (transactionID) => -new Date(transactions[transactionID])).slice(0, size);
```

Comment:
We shouldn't use underscore anymore in typescript files. We can just use the native `array.sort()` function for this.

Reply:
It is coming from Lodash, though. Is that still not ok?

Reply:
Yeah lodash and underscore are more or less the same. Guidelines are [here](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#migration-guidelines)